src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -263,15 +267,27 @@ void getChillerASHRAE205Input(EnergyPlusData &state)
         };
 
         thisChiller.SizFac = fields.at("sizing_factor").get<Real64>();
-        if (thisChiller.SizFac <= 0.0) thisChiller.SizFac = 1.0;
+        if (thisChiller.SizFac <= 0.0) {
+            thisChiller.SizFac = 1.0;
+        }
 
-        thisChiller.EvapVolFlowRate = fields.at("chilled_water_maximum_requested_flow_rate").get<Real64>();
-        if (thisChiller.EvapVolFlowRate == DataSizing::AutoSize) {
-            thisChiller.EvapVolFlowRateWasAutoSized = true;
+        {
+            auto tmpFlowRate = fields.at("chilled_water_maximum_requested_flow_rate");
+            if (tmpFlowRate == "Autosize") {
+                thisChiller.EvapVolFlowRate = DataSizing::AutoSize;
```

Comment:
Hmm, not sure why this field is causing so much trouble, we have autosized fields all over the place, but maybe not many places that are using the raw `fields.at` method instead of `getRealFieldValue` or `getObjectItem`. If I'm following what was done here, this needs at least a `samestring` to get a case-insensitive check for "autosize". Looking for an example to point to.

Reply:
We should not propogate this. Use getRealFieldValue instead. Plenty of examples.

Reply:
So, this method is similar to [`EIRPlantLoopHeatPump::processInputForEIRPLHP`](https://github.com/NREL/EnergyPlus/blob/02282dbcc4999c8bfb1655b5c489fff7c94631b9/src/EnergyPlus/PlantLoopHeatPumpEIR.cc#L883-L896). Hmm, no autosized example for that component. But I see this pattern used in UnitarySystem, so it must work. Never mind.

Reply:
UnitarySystem will no longer use this method after I get time to clean it out.

Reply:
@rraustad Don't bother with that. There are other epJSON input changes in the works.

Reply:
Sorry for all of the chatter here, but just for the record and my own sanity, I see the IdfParser forces all variations of autosize or autocalculate to "Autosize" [here](https://github.com/NREL/EnergyPlus/blob/02282dbcc4999c8bfb1655b5c489fff7c94631b9/src/EnergyPlus/InputProcessing/IdfParser.cc#L620-L635) where the "anyof" enum value is coming from the schema, e.g.:
```
                        "maximum_total_cooling_capacity": {
                            "note": "This field is ignored if Cooling Limit = NoLimit",
                            "units": "W",
                            "anyOf": [
                                {
                                    "type": "number",
                                    "minimum": 0.0
                                },
                                {
                                    "type": "string",
                                    "enum": [
                                        "Autosize"
                                    ]
                                }
                            ]
                        },
```
Learned something new today.

Reply:
Me too! 

BTW, @mjwitte, that's called a TIL (today I learned). That's how the kids talk these days. I only discovered this in the past few months myself. I think from @jmarrec. Now you learned two things: this thing about epJSON and TIL. The second one is the very meta TIL TIL. It's a good day.

Reply:
This exact change is one of those I pulled from Julien's branch, so I'm not sure what the best way to clean it up is (i.e. it will appear again).